modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/registry/NotificationUpgradeStepRegistrator.java

```diff
@@ -112,6 +112,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			new com.liferay.notification.internal.upgrade.v3_6_0.
 				NotificationQueueEntryUpgradeProcess(
 					_classNameLocalService, _resourceLocalService));
+
+		registry.register(
+			"3.6.0", "3.7.0",
+			new com.liferay.notification.internal.upgrade.v3_7_0.
+				NotificationQueueEntryUpgradeProcess());
```

Comment:
@murilostodolni I believe the upgrade class name should be `NotificationTemplateUpgradeProcess`

Reply:
@murilostodolni or `ResourcePermissionUpgradeProcess`.. I found some examples with this name, like in [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_0_1/ResourcePermissionUpgradeProcess.java)